cpp/arcticdb/storage/storages.hpp

```diff
@@ -21,6 +21,15 @@
 
 namespace arcticdb::storage {
 
+/*
+ * The Storages class abstracts over multiple physical stores and controls how ArcticDB read and write to multiple
+ * ones.
+ *
+ * Reasons for Storages to exist:
+ *  - Disaster recovery for Storages: if the first storage fails we can fall back to the second one, etc.
+ *  - Tiered storage: recent data goes to a fast, comparatively expensive storage and then is gradually moved
+ *  into a slower, cheaper one.
```

Comment:
These are the long term aims, its worth noting that neither feature is implemented right now

Reply:
Addressed by ccf5e71.